suite/tests/api/drdecode_x86.c

```diff
@@ -172,12 +172,12 @@ test_categories(void)
     /*  55 OP_mov_ld */
     instr = XINST_CREATE_load(GD, opnd_create_reg(DR_REG_XAX),
                               OPND_CREATE_MEMPTR(DR_REG_XAX, 42));
-    CHECK_CATEGORY(GD, instr, buf, DR_INSTR_CATEGORY_MOVE);
+    CHECK_CATEGORY(GD, instr, buf, (DR_INSTR_CATEGORY_MOVE | DR_INSTR_CATEGORY_LOAD));
```

Comment:
Hmm, is it weird that MOV is MOVE|LOAD yet POP is just LOAD?  Both are moving data around.  Should we have the logic that adds the LOAD also remove any MOVE?  Or should we add MOVE to all the PUSH and POP opcodes?  How do these operate on AArch64?  I'm guessing that there, there is no LOAD that is also MOVE?  So probably we should follow that convention?  (Except REP MOVS or a PUSH or POP which is both LOAD and STORE as they are moving from one memory slot to another and not loading/storing to a register endpoint??  Gray areas...)

Reply:
I think we need to remove move in this case. In my analysis I skip move in this case.
Thx, Kirill